src/main/webapp/app/layouts/navbar/navbar.component.ts

```diff
@@ -29,6 +33,7 @@ export class NavbarComponent implements OnInit {
   }
 
   ngOnInit(): void {
+    this.authSubscription = this.accountService.getAuthenticationState().subscribe(account => (this.account = account));
```

Comment:
itt szukseg van kesobb a valrtozora?

Reply:
Enélkül nem írja ki az éppen bejelentkezve lévő tanár vagy diák nevét. Az *ngIf használja navbar komponensébe (47 sor).